scripts/gen_vimdoc.py

```diff
@@ -68,7 +74,7 @@
 # Need a `nvim` that supports `-l`, try the local build
 nvim_path = Path(__file__).parent / "../build/bin/nvim"
 if nvim_path.exists():
-    nvim = str(nvim_path)
+    nvim = os.path.realpath(nvim_path)
```

Comment:
Since `nvim_path` is a `Path` object:

```suggestion
    nvim = nvim_path.resolve()
```